ui/js/filters.js

```diff
@@ -145,13 +138,6 @@ treeherder.filter('getRevisionUrl', ['thServiceDomain', function (thServiceDomai
     };
 }]);
 
-treeherder.filter('alertStatus', [
```

Comment:
Has been unused since it was added in 2015 (2acef3304199fe424c6f3b2585989c5f5fb83232)